library/ssl_srv.c

```diff
@@ -1306,7 +1306,7 @@ static int ssl_parse_client_hello( mbedtls_ssl_context *ssl )
         msg_len != mbedtls_ssl_hs_hdr_len( ssl ) + ( ( buf[2] << 8 ) | buf[3] ) )
     {
         MBEDTLS_SSL_DEBUG_MSG( 1, ( "bad client hello message" ) );
-        return( MBEDTLS_ERR_SSL_BAD_HS_CLIENT_HELLO );
+        return( MBEDTLS_ERR_SSL_DECODE_ERROR );
```

Comment:
Consider MBEDTLS_ERR_SSL_BAD_HS_CLIENT_HELLO ?

Reply:
This is being removed in this PR.